supplementary_materials/qwen2/requirements.txt

```
@@ -0,0 +1,7 @@
+nncf==2.10.0
+optimum-intel>=1.16.0
```

1.16.0 does not contain config for qwen2 if I right remember
```suggestion
optimum-intel>=1.17.0
```